packages/generator/package.json

```diff
@@ -23,6 +23,7 @@
     "build": "npm run renderTemplates && tsc && eslint dist --ext .ts,.js --quiet --fix",
     "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports/generator.xml",
     "test:debug": "nyc mocha \"test/**/*.test.ts\"",
+    "doc": "typedoc --mode modules --out ../../docs renderedTemplates/** --tsconfig tsconfig.json --external-modulemap  \".*/renderedTemplates/([\\w\\-_]+)/\"",
```

Comment:
Couple things here:
- Why docs at the root instead of just in this package?
- Do we need to check in the rendered docs?
  - If so, how do we keep them in sync with the code and make sure they're updated? Triggered by the build command?
  - If not, this should npm run renderTemplates